account/models.py

```diff
@@ -0,0 +1,113 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.db.models import Q
+import datetime
```

Comment:
Please split this files into two model files
account/models
       |-   __init__.py
       |- Profile.py
       |- Professional.py
       | - Client.py

Reply:
Did you mean 3? 
one for Profile, one for Professional and one for Client?